_changelog-files/2021/2021-06-29-google-search-console-v1-empty-response-error-message.md

```diff
@@ -0,0 +1,19 @@
+---
+title: "Google Search Console (v1) update: New empty response error exception"
+content-type: "changelog-entry"
+date: 2021-06-29
+entry-type: updated-feature
+entry-category: "integration"
+connection-id: "google-search-console"
+connection-version: 1
+pull-request: "https://github.com/singer-io/tap-google-search-console/pull/21"
+---
+{{ site.data.changelog.metadata.single-integration | flatify }}
+
+To improve the experience of our {{ this-connection.display_name }} integration, we've changed how we handle empty response errors.
+
+Instead of bypassing the error without an explanation, the integration will now raise an error and display it in the Extraction Logs:
+
+```shell
```

Comment:
I directly edited this because the suggestion feature wasn't working correctly for this change, but this is missing a closing set of ` ``` `

If you were checking this locally while you were working, this should've caused all the formatting to break below this entry on the changelog page.